src/main/resources/org/jenkinsci/plugins/zap/ZAPDriver/help-xmlAlertsFilter.html

```diff
@@ -0,0 +1 @@
+Select an XML Alert Filter File to use in the Context. The file must be stored in the Path provided for 'ZAP Settings'. </br></br><hr/></br>

```

Comment:
Change to:

`Select a ZAP alert filter to use in the context. The file must be stored in the Path provided for 'ZAP Settings'.`\

Changed the wording, notice that i removed the excess `</br></br><hr/></br>`